### PR TITLE
create project flow

### PR DESCRIPTION
**Pull request by EugeneTeu**
TSIA

Screenshot 2023-05-13 at 11 14 26 AM

https://github.com/upOnlyWithBCN/upOnly/assets/43085751/1c379e48-43be-473c-b027-07bd1d7a0623

# no test plan

### CODE REVIEW

File: src/components/Projects/ProjectCreate.tsx [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+                                        </NumberInput>
+                                    </StatNumber>
+                                </Stat>
+                                <Stat>
+                                    <StatLabel>Goal time</StatLabel>
+                                    <StatNumber>
+                                        <Input
+                                            type="date"
+                                            value={goal_time.toLocaleDateString(
+                                                'en-CA'
```
Comment (shaun97) [inline src/components/Projects/ProjectCreate.tsx:148]: why CA though?

Comment (EugeneTeu) [reply src/components/Projects/ProjectCreate.tsx:148]: Is to wrangle the date object into the format that the browser allows e.g dd-mm-yyyy lol found it off some random stack overflow post

Comment (EugeneTeu) [reply src/components/Projects/ProjectCreate.tsx:148]: So that I can pass a controlled (react) value to the input

**Review by shaun97 — APPROVED** [review]
lgtm

**Pull request merged**